app/channels.go

```diff
@@ -316,11 +316,10 @@ type hooksService struct {
 	ch *Channels
 }
 
-func (s *hooksService) RegisterHooks(productID string, hooks product.Hooks) error {
+func (s *hooksService) RegisterHooks(productID string, hooks any) error {
```

Comment:
Not so fast :P I am thinking of introducing generics in Q4. I want to wait a bit for things to stabilize. Totally fine with using it in mmctl/morph etc.

Reply:
I didn't consider this as generics TBH :p Isn't `any` just a [type alias](https://pkg.go.dev/builtin?utm_source=gopls#any) for `interface{}`?

Reply:
Right, that's fair. Since the server repo serves as a model for others to follow, we need to let everyone know that it's okay to start using `any` in the code. Do you want to queue an item in the dev meeting to let everyone know?